axelor-base/src/main/java/com/axelor/apps/base/service/administration/SequenceService.java

```diff
@@ -314,6 +314,17 @@ public String getDefaultTitle(Sequence sequence) {
     return item.getTitle();
   }
 
+  /**
+   * Get draft sequence number prefix.
+   *
+   * @return
+   */
+  public String getDraftPrefix() {
+    if (appBaseService.getAppBase().getDraftSuffix() != null)
```

Comment:
Use optional pattern, for example:
 `return Optional.ofNullable(appBaseService.getAppBase().getDraftSuffix()).orElse(DRAFT_PREFIX);`